addons/test_sale_product_configurators/static/tests/tours/product_configurator_edition_ui.js

```diff
@@ -40,13 +40,13 @@ registry.category("web_tour.tours").add('sale_product_configurator_edition_tour'
 }, {
     trigger: '.fa-pencil',
 }, {
-    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Customizable Desk"))>td:nth-child(2)>ul:nth-child(3)>li:nth-child(2)>div label:contains("Aluminium") ~ input:checked',
+    trigger: 'tr:has(.o_sale_product_configurator_name:contains("Customizable Desk"))>td:nth-child(2)>ul:nth-child(3)>li:nth-child(2)>div label:contains("Aluminium") > input',
```

Comment:
Why did you remove the `:checked` part of the trigger ? :think

Reply:
@Feyensv @Demesmaeker Done Sorry for that we forgot. Thanks